domain/src/test/java/uk/gov/hmcts/ccd/definition/store/domain/validation/casefield/CaseFieldEntityACLValidatorImplTest.java

```diff
@@ -51,21 +51,21 @@ public void setup() {
     @Test
     public void shouldHaveValidationErrorWhenUserNotFound() {
 
-        caseFieldUserRole.setUserRole(null);
-        caseFieldUserRole.setUserRoleId("nf_user_role_id");
+        caseFieldUserRole.setAccessProfile(null);
```

Comment:
rename `caseFieldUserRole`

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f

Reply:
Should probably still rename `caseFieldUserRole`